spec/venue_spec.rb

```diff
@@ -44,12 +44,40 @@
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
+
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+  describe '#over_capacity' do
+    it 'returns true if more patrons than capacity' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.over_capacity?
+      expect(venue.over_capacity?).to eq false
+      venue.add_patron('James')
+      venue.add_patron('Cat')
+      venue.over_capacity?
```

Comment:
You do not need this here

Reply:
So I don't need to run it before the assertion because it is executed in the assertion, correct?